pkg/destroy/powervs/powervs.go

```diff
@@ -751,6 +756,12 @@ func (o *ClusterUninstaller) ServiceInstanceNameToGUID(ctx context.Context, name
 				return "", fmt.Errorf("failed to get instance, response is: %v", response)
 			}
 
+			if resourceInstance.Type == nil {
+				o.Logger.Debugf("ServiceInstanceNameToGUID: type: nil")
+			} else {
+				o.Logger.Debugf("ServiceInstanceNameToGUID: type: %v", *resourceInstance.Type)
+			}
+
 			if resourceInstance.Type == nil || resourceInstance.GUID == nil {
 				continue
 			}
```

Comment:
```suggestion
			}
			o.Logger.Debugf("ServiceInstanceNameToGUID: type: %v", *resourceInstance.Type)
```

Reply:
The UI is not really making your proposed change clear to me.  I am now doing the same thing I did in pkg/destroy/powervs/serviceinstance.go.

Reply:
You're checking for `Type == nil` twice. I meant you can move the `else` body of the `if` you added after the existing check
```go
if resourceInstance.Type == nil || resourceInstance.GUID == nil {
    o.Logger.Debugf("ServiceInstanceNameToGUID: type: %v", resourceInstance.Type)
    continue
}
o.Logger.Debugf("ServiceInstanceNameToGUID: type: %v", *resourceInstance.Type)
```

Reply:
Thanks, pushed new version